docker-compose.yml

```diff
@@ -42,6 +49,8 @@ services:
       bahmni:
         aliases:
           - openmrs-1
+    ports:
+      - "8080:8080"
```

Comment:
@Fabiencha , here you have published the port **8080**, I guess for dev/debugging purposes.
However, in the **Bahmni Apps** and **Bahmni Config** section, you have removed the ports. It is either we publish them all or none at all.